assets/src/checkout/pages/BamboraPayPage.tsx

```diff
@@ -82,6 +115,21 @@ const bindEventListeners = (
       setCardErrors(errors => ({ ...errors, [event.field]: "" }));
     }
   });
+  checkout.on("brand", event => {
+    if (event.brand) {
```

Comment:
please redactor conditionals to return early. e.g. `if (!event.brand) { return; }`, and can also avoid using `else`.